### ISSUE TITLE
[Compatibility] Crying Obsidian portals dont work with BetterNether

### ISSUE THREAD

**Issue by nco2k** [Roundaround/mc-fabric-nicer-portals#11]
if you have [BetterNether](https://github.com/quiqueck/BetterNether) installed, then you cant create portals anymore that use Crying Obsidian. any chance to fix that?

**Comment by SplendidAlakey** [comment]
BetterNether has its own way of handling custom blocks for portals by utilizing tags: https://github.com/quiqueck/BetterNether/blob/1.20.3/src/main/generated/data/c/tags/blocks/nether_pframe.json

**Comment by Roundaround** [comment]
That's correct. In this case, they bypass the vanilla logic completely (whereas I sort of splice into it) so they by extension bypass my change as well. The best we could do would be to add crying obsidian to their new nether_pframe tag so that it gets picked up by BetterNether automatically.

You could do this today yourself with a datapack actually. https://minecraft.fandom.com/wiki/Tutorials/Creating_a_data_pack#Tags

I'm in the process of updating all my mods to 1.20.5/6 and then subsequently to 1.21, doing some long overdue feature work, bugfixes, and in some cases rewrites along the way. It's entirely possible I might just add this in there while I'm at it.
